packages/lesswrong/components/common/HashLink.tsx

```diff
@@ -0,0 +1,101 @@
+// Modified From: https://github.com/rafrex/react-router-hash-link/blob/master/src/index.js
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, NavLink } from 'react-router-dom';
```

Comment:
Should be going through our wrapper

Reply:
Nope, this component is itself used within our wrapper. So it can't import from our wrapper. 

I.e. HashLink itself is used in our definition of the Link component, so obviously it can't use the Link component that uses itself